### PR TITLE
Version management and execution environment + remove support for REFINEDVAR translation

### PR DESCRIPTION
**Pull request by cfsnook**
These changes sort out the versions and dependency based on API baseline tools.
We also decided to upgrade to Java 1.8

### CODE REVIEW

File: ac.soton.eventb.statemachines.animation.feature/feature.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <feature
       id="ac.soton.eventb.statemachines.animation.feature"
       label="%featureName"
-      version="2.3.1.qualifier"
+      version="2.4.0.qualifier"
       provider-name="%featureVendor"
       plugin="ac.soton.iumlb.branding"
```
Comment (tshoang) [inline ac.soton.eventb.statemachines.animation.feature/feature.xml]: This branding plug-in is missing. You should create a branding plug-in that use the same iUML-B logo.

**Review by tshoang — CHANGES_REQUESTED** [review]
Incorrect/missing branding plugin for this feature.

File: ac.soton.eventb.statemachines.animation.feature/feature.xml (hunk 1 above)
Comment (cfsnook) [reply ac.soton.eventb.statemachines.animation.feature/feature.xml]: Branding plugin was not missing - it was a common branding plugin used by all iUML-B features. (see iUML-B utils. git)
However, we decided to adopt separate branding plugins per feature

**Comment by cfsnook** [comment]
I think this is now resolved and i have made it .released 
Can you approve this one?

**Comment by cfsnook** [comment]
Looks like this review didn't get approved previously. 
I have now made another change to export a package to API

**Review by tshoang — APPROVED** [review]
Looks good to me.

**Pull request merged**